OMEdit/OMEditGUI/Animation/AbstractAnimationWindow.cpp

```diff
@@ -207,9 +213,138 @@ void AbstractAnimationWindow::createActions()
   mpRotateCameraRightAction = new QAction(QIcon(":/Resources/icons/rotateCameraRight.svg"), tr("Rotate Right"), this);
   mpRotateCameraRightAction->setStatusTip(tr("Rotates the camera right"));
   connect(mpRotateCameraRightAction, SIGNAL(triggered()), this, SLOT(rotateCameraRight()));
+  //interactive control action
+  mpInteractiveControlAction = mpAnimationParameterDockerWidget->toggleViewAction();
+  mpInteractiveControlAction->setIcon(QIcon(":/Resources/icons/control-panel.svg"));
+  mpInteractiveControlAction->setText(tr("interactive control"));
+  mpInteractiveControlAction->setStatusTip(tr("Opens the interactive control panel"));
+  connect(mpInteractiveControlAction, SIGNAL(triggered()), this, SLOT(showOrHideInteractionPanel()));
```

Comment:
You don't need this since the above line `mpInteractiveControlAction = mpAnimationParameterDockerWidget->toggleViewAction();` automatically does that. So you can get rid of `showOrHideInteractionPanel` slot .